CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ All notable changes to this project will be documented in this file.
 ### New Features
 - Call `IexecHubContract#viewDeal` instead of legacy ABI in `IexecHubAbstractService#getChainDeal`. (#24)
 - Representations of on-chain object are now immutable. (#24 #25 #26)
+### Bug Fixes
+- Add jackson annotations for `ChainReceipt` json deserialization. (#30)
```

Comment:
When the bug is introduced and fixed in the same version, shouldn't we merge both changelog lines?
:arrow_down: 
```suggestion
- Representations of on-chain object are now immutable. (#24 #25 #26 #30)
```